README.md

```diff
@@ -26,35 +26,55 @@
 
 ## Introduction
 
-Lets goooo - Next.js 14, Prisma, Planetscale, Auth.js, Resend, React Email, Shadcn/ui, and Stripe.
+Lets goooo - Next.js 14, Turborepo, Prisma, Planetscale, Auth.js, Resend, React Email, Shadcn/ui, and Stripe.
 <br/>
 All seamlessly integrated with the Projectx to accelerate the development.
 
+## Directory Structure
+
+ProjectX is a monorepo managed by [Turborepo](https://turbo.fish/). The monorepo is split between `apps` and `packages` directories.
+
+- **Apps** are the Next.js apps that are deployed to Vercel (this is where most development is done).
+- **Packages** are the shared packages that are used by the apps (e.g. `@projectx/components`)
+
+````
+
```

Comment:
As Images Speak Louder Than Words.

How about we add a directory tree in Readme? which tells clearly where all folders belong.
E.g
https://github.com/kriasoft/Folder-Structure-Conventions/tree/master?tab=readme-ov-file#a-typical-top-level-directory-layout

Reply:
Also please change turborepo link to https://turbo.build/repo

Reply:
Yes! That sounds like a plan!